tv2.html

```diff
@@ -0,0 +1,283 @@
+---
+layout: default
+permalink: /tv2
+---
+<section class="bg-tan-light soft-sides soft-top">
+  <div class="soft-sides soft-top">
+    <div class="container soft-top hard-sides">
+      <div class="col-md-8">
+        <div class="latest-message">
+          <div class="latest-message-background soft-half-bottom push-bottom">
+            <div class="embed-container">
+              <a href="#item" data-automation-id="message-video">
```

Comment:
I changed `href="{{ item | media_url }}"` on this line to `href="#item"` so the build wouldn't fail